src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -1100,13 +1100,19 @@ def rescan_progress(wallet_alias):
 @wallets_endpoint.route("/wallet/<wallet_alias>/get_label", methods=["POST"])
 @login_required
 def get_label(wallet_alias):
-    wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
-    address = request.form.get("address", "")
-    label = wallet.getlabel(address)
-    return {
-        "address": address,
-        "label": label,
-    }
+    try:
+        wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
+        address = request.form.get("address", "")
+        label = wallet.getlabel(address)
+        return {
+            "address": address,
+            "label": label,
+        }
+    except Exception as e:
```

Comment:
Can you please log the exception with e.g. handle_exception? Rule of thumb should imho be: If you threw the exception yourself, you probably don't neeed a stacktrace in the logs. But if you catch an exception, you probably don't know much about the error, so please then make sure to report it in the logs.

Reply:
Added